.env.example

```diff
@@ -1,26 +1,30 @@
-# bancho.py supports both INET and UNIX sockets
-# INET: provide an IP for SERVER_ADDR, and a port for SERVER_PORT
-# UNIX: provide a filesystem path for SERVER_ADDR, and no SERVER_PORT
-SERVER_ADDR=/tmp/bancho.sock
-SERVER_PORT=
+# for nginx reverse proxy to work through the containers, the bancho
+# server must expose itself on port 80 to be accessed on http://bancho.
+SERVER_ADDR=0.0.0.0
+SERVER_PORT=80
 
+HOST_PORT=443
+
+# XXX: change your db credentials
 DB_USER=cmyui
 DB_PASS=lol123
-DB_HOST=localhost
+DB_NAME=banchopy
+
+DB_HOST=bpy_mysql
```

Comment:
generally just "mysql" and "redis" are standard for these, no need to prefix them imo

Reply:
will be applied in next commit